demisto_sdk/commands/validate/validators/PA_validators/PA101_is_version_match_rn.py

```diff
@@ -0,0 +1,40 @@
+from __future__ import annotations
+
+from typing import Iterable, List
+
+from packaging.version import Version
+
+from demisto_sdk.commands.content_graph.objects.pack import Pack
+from demisto_sdk.commands.validate.validators.base_validator import (
+    BaseValidator,
+    ValidationResult,
+)
+
+ContentTypes = Pack
+
+
+class IsVersionMatchRnValidator(BaseValidator[ContentTypes]):
+    error_code = "PA101"
+    description = "Validate that the version mentioned in the Pack metadata matches the latest RN version."
+    error_message = "The currentVersion in the metadata ({0}) doesn't match the latest rn version ({1})."
+    related_field = "currentVersion"
+
+    def is_valid(self, content_items: Iterable[ContentTypes]) -> List[ValidationResult]:
+        return [
+            ValidationResult(
+                validator=self,
+                message=self.error_message.format(
+                    content_item.current_version,
+                    (content_item.latest_rn_version or "none"),
```

Comment:
or "none" can u explain?

Reply:
It's for the error msg, to make it more informational.
"The currentVersion in the metadata ({0}) doesn't match the latest rn version ({1})."
will yield
"The currentVersion in the metadata ({0}) doesn't match the latest rn version (none)."